Added all device support options (#2652)
balena-os/meta-balena: contributing-device-support.md

# Customer Board Support

> There are several options for onboarding the hardware onto balenaOS:

Reviewer: ```suggestion
There are several ways for enabling balenaOS support for your hardware:
```

> 1. You can find another board that would work for your case that is supported in our [supported device list](https://www.balena.io/docs/reference/hardware/devices/).

Reviewer: ```suggestion
1. Checking out if balenaOS doesn't already provide support for your board through our [supported devices list](https://www.balena.io/docs/reference/hardware/devices/).
```

This guide is probably in a place where they are checking out the custom image build process. So this point can be better phrased as a check that they should make.

> 2. You can consider our [Custom Device Support (CDS)](https://www.balena.io/pricing/#custom-device-support), which is a paid service during which we onboard the device for you and maintain it for a monthly fee. [Here](https://drive.google.com/file/d/17-DTeXkflkzW8Qduj48ut7JNUTWqiRv-/view?usp=sharing) is an overview including pricing.

Reviewer: ```suggestion
2. Consider our [Custom Device Support](https://www.balena.io/pricing/#custom-device-support) (CDS) service. This is a paid service where we onboard the device to balenaOS for you and maintain it for a monthly fee. Checkout out the [pricing for CDS](LINK) and how to get the process started for you. 
```

`Here` is not an accessible word for hyperlinks for folks reading our documentation using Read Aloud software. Added a suggestion for the sentence. 
Drive links are private and can easily deprecate. Please add a public link instead as we talked about.

> 3. If you have folks on your team who are familiar with Yocto, then you could build and maintain it yourselves using our customer board support (CBS) documentation. Follow along below!

Reviewer: ```suggestion
3. Building and maintaining balenaOS yourselves using our Customer Board Support (CBS) documentation. This will require knowledge of the Yocto project and familiarity with tools used to build custom images. Follow along the documentation for CBS below: 
```

> **Pre-requisites:** a [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible with the Yocto releases balenaOS supports.

Reviewer: ```suggestion
## Pre-requisites

A [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible with the Yocto releases balenaOS supports.
```

Since the para above becomes intro, I feel like we need a pre-requistes para
